src/status_im/transport/message/core.cljs

```diff
@@ -26,13 +26,19 @@
   dedup-id is passed by status-go and is used to deduplicate messages at that layer.
   Once a message has been successfuly processed, that id needs to be sent back
   in order to stop receiving that message"
-  [cofx now-in-s filter-chat-id message]
+  [cofx now-in-s filter-chat-id message-js]
   (let [blocked-contacts (get-in cofx [:db :contacts/blocked] #{})
-        {{:keys [payload timestamp]} :message
-         metadata :metadata
-         raw-payload :raw-payload} (add-raw-payload message)
+        payload (.-payload message-js)
+        timestamp (.-timestamp (.-message message-js))
+        metadata-js (.-metadata message-js)
+        metadata {:author {:publicKey (.-publicKey (.-author metadata-js))
```

Comment:
https://github.com/mfikes/cljs-bean probably could be useful here, although if we are moving to go anyway...

Reply:
Yes, I think we should be using it eventually, as at some point I think we will pass the parsed JSON messages from status-go, and then it would come handy